pulsar-metadata/src/main/java/org/apache/pulsar/metadata/impl/AbstractMetadataStore.java

```diff
@@ -126,6 +131,9 @@ public CompletableFuture<Boolean> asyncReload(String key, Boolean oldValue,
                         }
                     }
                 });
+
+        this.metadataStoreName = poolName + "-" + ID.getAndIncrement();
+        this.metadataStoreStats = new MetadataStoreStats(metadataStoreName);
```

Comment:
`this.` seem redundant here. It's relevant to all places in this PR.

Reply:
personal code style. and Pulsar checkstyle doesn't requires we can't or we must add a `this.` prefix